tests/bcl-test/templates/common/TestRunner.NUnit/NUnitTestRunner.cs

```diff
@@ -35,7 +36,9 @@ public NUnitTestRunner (LogWriter logger) : base (logger)
 			builderSettings = new Dictionary<string, object> (StringComparer.OrdinalIgnoreCase);
 		}
 
-		public override void Run (IEnumerable<TestAssemblyInfo> testAssemblies)
+#pragma warning disable 1998
+		public override async Task Run (IEnumerable<TestAssemblyInfo> testAssemblies)
```

Comment:
async + param + function name = normal declaration for asynchronous functions in C#?

Reply:
In this case, @baulig is defining a async method, that is in reality, sync. He does that so that both runners have the same interface. That will make the c# compiler throw a warning, and we want it to be muted since we know what we are doing.